src/Storages/MergeTree/MergeTreeIndexText.h

```diff
@@ -297,9 +304,30 @@ struct MergeTreeIndexTextGranuleBuilder
     std::unique_ptr<Arena> arena;
 };
 
+class MergeTreePreprocessor
+{
+public:
+    MergeTreePreprocessor(const String & expression, const IndexDescription & index_description);
+    std::pair<ColumnPtr,size_t> processColumn(const ColumnWithTypeAndName & index_column_with_type_and_name, size_t start_row, size_t n_rows) const;
```

Comment:
I think method `processColumn` deserves a comment. It is not immediately clear what the second returned value (size_t) represents.

Reply:
Solved via a86e3a3e1403a4bcfee6c